comp/core/workloadmeta/collectors/internal/containerd/image_sbom_trivy.go

```diff
@@ -55,8 +55,11 @@ func (c *collector) startSBOMCollection(ctx context.Context) error {
 				close(resultChan)
 				return
 
-			case eventBundle := <-imgEventsCh:
-				close(eventBundle.Ch)
+			case eventBundle, ok := <-imgEventsCh:
```

Comment:
can be good to add a comment that explain the utility of checking `ok`

Reply:
I think it should be in the doc comment of the `Unsubscribe` function

Reply:
you can put it in both place